packages/amplication-server/src/core/build/build.controller.ts

```diff
@@ -82,18 +83,18 @@ export class BuildController {
   @EventPattern(
     EnvironmentVariables.instance.get(Env.CREATE_PR_FAILURE_TOPIC, true)
   )
-  async onCreatePRFailure(@Payload() message: KafkaMessage): Promise<void> {
+  async onCreatePRFailure(@Payload() message: CreatePRFailure): Promise<void> {
     try {
-      const args = plainToInstance(CreatePRFailure, message.value);
+      const args = plainToInstance(CreatePRFailure, message);
```

Comment:
is `plainToInstance` still needed? 
Nestjs will attempt to parse the message to JSON and the right type here